docs/contribute.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Primer
+
+There are several ways you can contribute to and participate in Primer:
+
+- Participate in discussions
+- Submit an issue
+- Improve the documentation
+- Close issues
+- Propose a new component or UI pattern
+- Questions and support
+
+Please note that all contributions to Primer need to follow our code of conduct.
+
+## Participate in discussions
+
+Discussions about improvements and upcoming work happen mostly in issues and in the #design-system Slack channel. These are the repos you should watch if you’re interested in getting involved:
+
+- Design
+- Primer CSS
+- Primer Components (React)
+- Octicons
+- ...
```

Comment:
Which repos should we highlight here?

Reply:
wouldn't this be any repo in github.com/primer that's open source? + linking to them